app/views/webhook_deliveries/show.html.haml

```diff
@@ -0,0 +1,42 @@
+%h1 Webhook Delivery Details
```

Comment:
More detailed view of this page:

<img width="895" alt="Screenshot 2024-10-08 at 4 14 54 PM" src="https://github.com/user-attachments/assets/70933508-dca3-4324-b8e7-57b9bfabe928">

Reply:
Added some early pings [@Mitch-Henson @jonallured ] for feedback on layout + data presentation

Reply:
I think it looks great! It shows all the information needed